### PR TITLE
Submarine 1095. Incorporate Click CLI in SDK

### PR DESCRIPTION
**Pull request by atosystem**
### What is this PR for?
Introduce click module to submarine SDK.

Allowing the followings commands
```
submarine sandbox start # create submarine sandbox 
submarine sandbox start --version 0.6.0 
submarine sandbox delete # delete submarine sandbox 

submarine get experiment <id>
submarine get notebook <id>
submarine get environment <id>

submarine list experiment 
submarine list notebook
submarine list environment

submarine delete experiment <id>
submarine delete notebook <id> 
submarine delete environment <id>
```

### What type of PR is it?
[Feature]

### Todos
* [ ] - Connect backend API
* [ ] - Write docs for CLI

### What is the Jira issue?
[SUBMARINE-1095](https://issues.apache.org/jira/browse/SUBMARINE-1095)

### How should this be tested?

python unit tests are implemented for command line testing

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? Yes

### CODE REVIEW

File: submarine-sdk/pysubmarine/submarine/cli/__init__.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# )
+
+# # import models into sdk package
+# from submarine.experiment.models.code_spec import CodeSpec
+# from submarine.experiment.models.environment_spec import EnvironmentSpec
+# from submarine.experiment.models.experiment_meta import ExperimentMeta
+# from submarine.experiment.models.experiment_spec import ExperimentSpec
+# from submarine.experiment.models.experiment_task_spec import ExperimentTaskSpec
+# from submarine.experiment.models.json_response import JsonResponse
+# from submarine.experiment.models.kernel_spec import KernelSpec
```
Comment (KUAN-HSUN-LI) [inline submarine-sdk/pysubmarine/submarine/cli/__init__.py:52]: Remove these comments?

**Review by KUAN-HSUN-LI — COMMENTED** [review]
Thanks for your contribution. There is one comment.

File: submarine-sdk/pysubmarine/submarine/cli/__init__.py (hunk 1 above)
Comment (atosystem) [reply submarine-sdk/pysubmarine/submarine/cli/__init__.py:52]: OK!

**Review by KUAN-HSUN-LI — APPROVED** [review]
LGTM

**Pull request closed without merging**